src/shared/helper.js

```diff
@@ -35,7 +35,13 @@ export const getDate = (day) => {
     return date;
   };
   let date = new Date();
-  return date.addDays(day).toISOString().split("T")[0];
+  return date
+    .addDays(day)
+    .toLocaleString("en-GB", { timeZone: "UTC" })
```

Comment:
This change makes no difference to the existing code. Please check
![image](https://user-images.githubusercontent.com/18077542/124753782-0b49f380-df47-11eb-93d4-ae3324603c8d.png)